src/modules/check.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable  import/no-cycle */
```

Comment:
- [optional] Please, it would be great if you can read one or two articles online on how to resolve this linter issue, [this article can be a good starting point](https://stackoverflow.com/questions/67855711/eslint-error-dependency-cycle-detected-import-no-cycle). Disabling linters at times is not the best. ℹ️

Reply:
Thank you so much.